params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
I'm kinda thinking we just make you restart roscore.  Of course, if you know you're using the same nodes then you don't have to, but that's kinda a hidden bonus